.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Fixes # 
+
+### Checklist
+- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar).
+- [ ] I have read the [Contribution Guide](CONTRIBUTING.md) and my PR follows them.
+- [ ] I have discussed these changes via issue.
+- [ ] I have made my changes on a new branch.
+- [ ] My code follows the code style of this project.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+
+
+### Short description of what this PR does:
+- 
+- 
+
+<p>Please enter each Issue number you are resolving in your PR after one of the following words [Fixes, Closes, Resolves]. This will auto-link these issues and close them when this PR is merged!</p>
```

Comment:
I think this should be a comment so that it's not visible when we're sending the PR. As it is, most of us will keep the text, only a few will delete it.